### DISCUSSION TITLE
蓝灯跑路了吗，为啥充了钱两周不能开通会员？

### DISCUSSION THREAD

**Discussion by hujb2000** [getlantern/lantern-forum-cn#807] [General]
蓝灯跑路了吗，为啥充了钱两周不能开通会员？

**Comment by JMaster1937** [comment]
根本不回，没人管

**Comment by MaxMKMax** [comment]
@hujb2000 我们是不会跑路的! 请您提供我支付单据和账号邮箱. 我帮您提交问题弄账号升级.

**Reply by hujb2000** [reply]
@Mike777Khoo 
![1](https://github.com/user-attachments/assets/9aaaecfe-03df-4028-99d4-6b4218d6cf89)
![2](https://github.com/user-attachments/assets/bba32b22-296d-4681-bfd4-eebee4d494ee)
Account Email:  hujb2000@163.com

Thank you

**Reply by MaxMKMax** [reply]
@hujb2000 
好的. 已经帮您提交问题上去了.

**Reply by hujb2000** [reply]
@Mike777Khoo ，谢谢，问题主体已解决。 现在专业版到期日显示为:  8/28/2027，但是奖励的4个月没有加进去。

**Reply by MaxMKMax** [reply]
@hujb2000 您好. 不客气呢. 好的, 我帮您问看那 4 个月的奖励让他们帮您加上去. 谢谢您支持蓝灯! 

还有啊, 我们是不会跑路的啊!!! 我们是用真诚对待客户, 希望提供你们最好的服务.
